src/BackwardCompatiblityTests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,19 @@
+using System.Reflection;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("Axe.Windows.BackwardCompatiblityTest")]
```

Comment:
Misspeling
```suggestion
[assembly: AssemblyTitle("Axe.Windows.BackwardCompatibilityTest")]
```

Reply:
Renamed to Axe.Windows.OldFileVersionCompatibilityTests